### PR TITLE
MySQL: account table restructure

### PR DESCRIPTION
**Pull request by Viste**
##### Target branch(es): Master

### CODE REVIEW

File: data/sql/updates/pending_db_auth/rev_1546541118941824046.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO version_db_auth (`sql_rev`) VALUES ('1546541118941824046');
+
+ALTER TABLE `account` CHANGE `email` `email` VARCHAR(255) NOT NULL DEFAULT '';
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_auth/rev_1546541118941824046.sql]: missing new line(s) at the end of file(s)

Comment (FrancescoBorzi) [reply data/sql/updates/pending_db_auth/rev_1546541118941824046.sql]: also you can put those changes in one single sql file since it's the same database

Comment (Viste) [reply data/sql/updates/pending_db_auth/rev_1546541118941824046.sql]: i separate them to be more clean

Comment (BarbzYHOOL) [reply data/sql/updates/pending_db_auth/rev_1546541118941824046.sql]: put them in the same file please, it's more logic for our case (and with the "insert into revision" )

File: data/sql/updates/pending_db_auth/rev_1546540718864817294.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+INSERT INTO version_db_auth (`sql_rev`) VALUES ('1546540718864817294');
+
+ALTER TABLE `account` CHANGE `last_login` `last_login` TIMESTAMP NULL;
```
Comment (Meltie2013) [inline data/sql/updates/pending_db_auth/rev_1546540718864817294.sql]: this should be like this
```alter table account change last_login last_login timestamp null default null```
it's a more clean way of doing it.

**Comment by wetbrownsauce** [comment]
Not seeing any issues from this in game so no complaints from me. I agree with the proposed change by meltie though.

File: data/sql/updates/pending_db_auth/rev_1546540718864817294.sql (hunk 2 above)
Comment (Viste) [reply data/sql/updates/pending_db_auth/rev_1546540718864817294.sql]: lol you make colum null by default i'm not. no clean or dirty

Comment (Meltie2013) [reply data/sql/updates/pending_db_auth/rev_1546540718864817294.sql]: Doing it the way you have it in this PR, will just repeat the issue you're trying to close. So your PR is just useless without a proper method of fixing it.

Comment (BarbzYHOOL) [reply data/sql/updates/pending_db_auth/rev_1546540718864817294.sql]: Yes, default should be Null I guess

Comment (FrancescoBorzi) [reply data/sql/updates/pending_db_auth/rev_1546540718864817294.sql]: `last_login` timestamp NULL DEFAULT NULL,`

**Pull request merged**